spec/index.md

```diff
@@ -6477,7 +6476,7 @@ The pseudo-code below does *not* explicitly enforce the restrictions of which im
       ) < 0 or system_cannot_do_this_call_now()
       then
         discard_pending_call<es>()
-        return 1
+        return <implementation-specific>
```

Comment:
I guess this should be non-zero, but I understand it's difficult to specify given the convention we adopted elsewhere.

Reply:
I don't think the error can be implementation specific as this happens in the deterministic execution?

Reply:
sure, but we don't want to specify it (just like the amount of cycles consumed which also happens in the deterministic execution, but we mark it as implementation-specific since we don't cover it in this specification)